scala-package/assembly/pom.xml

```diff
@@ -26,7 +26,7 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-  <artifactId>mxnet-full_2.11</artifactId>
+  <artifactId>mxnet-full</artifactId>
```

Comment:
Is there a reason you are removing the _{scala.binary.version} from the internal artifactId? This could end up breaking other users

Reply:
This is an artifactId in the Maven _reactor_ scope. Concrete Scala version suffix adds on deploy/install phase. It means that artifact published to the Central will have Scala version suffix. See `deploy/pom.xml`

Reply:
When install scala-package locally, `~/.m2/repository` will contain artifact without suffix. I sure that it advanced usage option, and user know what he or she doing. At least he or she explicitly sets profile with Scala version.